lib/src/day_view/day_view.dart

```diff
@@ -360,6 +349,77 @@ class DayViewState<T> extends State<DayView<T>> {
     }
   }
 
+  void _updateViewDimensions() {
+    _width = widget.width ?? MediaQuery.of(context).size.width;
+    assert(_width != 0, "Calendar width can not be 0.");
```

Comment:
should assert post class constructor so it can prevent user from future breaking changes.